internal/commands/iam/groups/iam/iam.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package iam
+
+import (
+	"github.com/hashicorp/hcp/internal/pkg/cmd"
+	"github.com/hashicorp/hcp/internal/pkg/heredoc"
+)
+
+func NewCmdIAM(ctx *cmd.Context) *cmd.Command {
+	cmd := &cmd.Command{
+		Name:      "iam",
+		ShortHelp: "Manage a group's IAM policy.",
+		LongHelp: heredoc.New(ctx.IO).Must(`
+		The {{ template "mdCodeOrBold" "hcp iam groups iam" }} command group lets you
+		manage a group's IAM Policy.
```

Comment:
Just a note for later. Once we have all this implemented, it would be good to add examples here for how to make a user a group manager.

Reply:
That makes sense! I'll add a note to add that in for later